articles/setup-misskey-by-kubernetes.md

```
@@ -0,0 +1,712 @@
+
+- ArgoCD
+  - GitOps による k8s CI/CD ツール
+- ExternalSecretsOperator
+  - k8s Secret を外部ストアに保存できる Custom Operator
+
+そのため、 Misskey のマニフェスト構成に ArgoCD と ExternalSecretsOperator を利用しています。
+よって、後続の Manifest にはこれらが要素として出てきます。
+
+# 構成図
+
+![](https://storage.googleapis.com/zenn-user-upload/4af1d373e0d8-20241102.png)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)